servers/rendering/renderer_rd/forward_clustered/render_forward_clustered.cpp

```diff
@@ -200,20 +201,25 @@ void RenderForwardClustered::RenderBufferDataForwardClustered::configure(RID p_c
 			fb.push_back(p_color_buffer);
 			fb.push_back(depth);
 
-			color_fb = RD::get_singleton()->framebuffer_create(fb);
+			color_fb = RD::get_singleton()->framebuffer_create(fb, RenderingDevice::INVALID_ID, views);
 		}
 		{
 			Vector<RID> fb;
 			fb.push_back(depth);
 
-			depth_fb = RD::get_singleton()->framebuffer_create(fb);
+			depth_fb = RD::get_singleton()->framebuffer_create(fb, RenderingDevice::INVALID_ID, views);
 		}
 	} else {
 		RD::TextureFormat tf;
+		if (views > 1) {
+			tf.texture_type = RD::TEXTURE_TYPE_2D_ARRAY;
+		} else {
+			tf.texture_type = RD::TEXTURE_TYPE_2D;
+		}
 		tf.format = RD::DATA_FORMAT_R16G16B16A16_SFLOAT;
 		tf.width = p_width;
 		tf.height = p_height;
-		tf.texture_type = RD::TEXTURE_TYPE_2D;
+		tf.array_layers = views;
```

Comment:
likewise the same here layers to layer_count

Reply:
Hmm, in this case I disagree.. it is view count here too, the problem is that array layers is used as the target for the different views. Else we'll just have the view_count turn into layer_count where it is actually reflecting view count.

Reply:
I've added a remark that we need a layer for every view wherever we do this assignment.